lib/gosu/compat.rb

```diff
@@ -0,0 +1,187 @@
+# This whole file is about backward compatibility
+
+# First - define some methods to tell the world about their old code
+module Gosu
+  module Deprecate
```

Comment:
I actually moved the code a few time forth and back between a seperate modul and Gosu itself - not sure which version I like more. The final decision was based on the fact, that I wanted the const_missing hook easily available in every class (if needed) and I felt quite uncomfortable extending the Gosu-module, as there is quite some stuff defined. As there are currently only (deprecated) constants und Gosu it would work I think.